core/templates/core/index.html

```diff
@@ -3,5 +3,16 @@
 {% block title %}Strona Główna{% endblock %}
 
 {% block content %}
-    <p class="text-3xl font-bold underline text-red-600">Gwardia Czapli</p>
+    <div class="flex flex-col flex-nowrap items-center">
+        <h1 class="text-4xl font-bold underline text-red-600 mt-10 mb-16">Gwardia Czapli</h1>
+        <div class="flex flex-col flex-nowrap items-start mr-auto ml-16">
+            <h2 class="text-3xl -ml-4">Gwardia Hub website</h2>
```

Comment:
This can be centered

Reply:
I think article header should be left-aligned, it just looks better, but I will change this h2 to Project goal